### PR TITLE
Add LC

### PR DESCRIPTION
**Pull request by raynamharris**
This PR 
* adds a very history and some links to LC
* reorganizes the similarities and differences section a little to accomodate LC
* removes the DC/SWC venn diagram
* adds a brief history image ([also on Twitter](https://twitter.com/raynamharris/status/1062516187874189312))
* add links to the lesson pages
* some minor tweaking of headers

### CODE REVIEW

**Comment by raynamharris** [comment]
someone should confirm in the language about projects and lesson orgs  is correct (as per https://github.com/carpentries/instructor-training/issues/639)

File: _episodes/21-carpentries.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ block: "The Carpentries"
 teaching: 30
 exercises: 45
 questions:
-- "How are Software, Data, and Library Carpentry organized and run?"
+- "How is The Carpentries organized and run?"
+- "What is the difference between SWC, DC, and SWC workshops?"
```
Comment (amyehodge) [inline _episodes/21-carpentries.md]: Should read "SWC, DC, and LC workshops?"

File: _episodes/21-carpentries.md [hunk 2]
```diff
@@ -14,40 +15,53 @@ keypoints:
...
-*   a focus on computational skills.
+This section provides some background on 
+[The Carpentries]({{ site.carpentries_site }}) projects and information
+about how to get involved.
+
+## A Brief History
+
+[Software Carpentry]({{ site.swc_site }}) was founded in 1998 
+with the mission of teaching lab skills for research computing. 
+[Data Carpentry]({{ site.dc_site }}) was founded in 2014 with the mission of building communities teaching universial data literacy.
```
Comment (amyehodge) [inline _episodes/21-carpentries.md]: typo -- universial should be universal

File: _episodes/21-carpentries.md [hunk 3]
```diff
@@ -14,40 +15,53 @@ keypoints:
 
 ---
 
-In becoming a Carpentry instructor,
+In becoming a certified [Carpentry instructor](https://carpentries.org/instructors/),
```
Comment (amyehodge) [inline _episodes/21-carpentries.md]: I believe this should be "Carpentries instructor" not "Carpentry instructor"

File: _episodes/21-carpentries.md [hunk 4]
```diff
@@ -14,40 +15,53 @@ keypoints:
...
+with the mission of teaching lab skills for research computing. 
+[Data Carpentry]({{ site.dc_site }}) was founded in 2014 with the mission of building communities teaching universial data literacy. 
+Also in 2014, [Library Carpentry](({{ site.lc_site }})) was founded with the mission of teaching data skills to people working in library- and information-related roles.
+
+One January 1, 2018, Software Carpentry and Data Carpentry merged 
+their projects to form a new project called
+[The Carpentries]({{ site.carpentries_site }})
+under the fiscal sponsorship of 
+[Community Initiatives](https://communityin.org/)
+On November 1, 2018, The Carpentries Executive Council
```
Comment (amyehodge) [inline _episodes/21-carpentries.md]: "One" should be "On"
After this sentence I would add in -- "Within this new organization, Software Carpentry and Data Carpentry retain their individual identities as Lesson Programs of the Carpentries."

File: _episodes/21-carpentries.md [hunk 5]
```diff
@@ -14,40 +15,53 @@ keypoints:
...
+Also in 2014, [Library Carpentry](({{ site.lc_site }})) was founded with the mission of teaching data skills to people working in library- and information-related roles.
+
+One January 1, 2018, Software Carpentry and Data Carpentry merged 
+their projects to form a new project called
+[The Carpentries]({{ site.carpentries_site }})
+under the fiscal sponsorship of 
+[Community Initiatives](https://communityin.org/)
+On November 1, 2018, The Carpentries Executive Council 
+approved [Library Carpentry](({{ site.lc_site }})) 
+as an official Lesson Program.
```
Comment (amyehodge) [inline _episodes/21-carpentries.md]: I would modify this to "... approved Library Carpentry as the third official Lesson Program of the Carpentries."

File: _episodes/21-carpentries.md [hunk 6]
```diff
@@ -118,9 +140,9 @@ under a permissive [open license]({{ page.root }}/LICENSE).
...
 provided you cite the original source.
 
-### Using the Names
+### Using the Names and Logos
 
-The names "Software Carpentry" and "Data Carpentry"
+The names "Software Carpentry", "Data Carpentry" and "Library Carpentry"
 and their respective logos
 are all trademarked.
 You may only call a workshop a Software Carpentry or Data Carpentry workshop if:
```
Comment (amyehodge) [inline _episodes/21-carpentries.md]: Likely this should read "Software Carpentry, Data Carpentry, or Library Carpentry workshop if:"

File: _episodes/21-carpentries.md [hunk 7]
```diff
@@ -130,13 +152,22 @@ You may only call a workshop a Software Carpentry or Data Carpentry workshop if:
...
 and a programming language (e.g. R or Python).
 
-A Data Carpentry workshop must include a Data Carpentry lesson on data organization and three
+A Data Carpentry workshop must include a Data Carpentry [lesson](https://datacarpentry.org/lessons/) 
+on data organization and three
 other modules in the same domain from the Data Carpentry curriculum.
+Published curricula include the Ecology, Genomics, Social Science and Geospatial workshop materials.
+Additionally, Astronomy, Digital humanities, Economics and Image analysis curriculum are under development. 
+
+A Library Carpentry [core curriculum](https://librarycarpentry.github.io/test/lessons/)
```
Comment (amyehodge) [inline _episodes/21-carpentries.md]: should read "A Library Carpentry workshop...."

File: _episodes/21-carpentries.md [hunk 8]
```diff
@@ -130,13 +152,22 @@ You may only call a workshop a Software Carpentry or Data Carpentry workshop if:
...
+A Data Carpentry workshop must include a Data Carpentry [lesson](https://datacarpentry.org/lessons/) 
+on data organization and three
 other modules in the same domain from the Data Carpentry curriculum.
+Published curricula include the Ecology, Genomics, Social Science and Geospatial workshop materials.
+Additionally, Astronomy, Digital humanities, Economics and Image analysis curriculum are under development. 
+
+A Library Carpentry [core curriculum](https://librarycarpentry.github.io/test/lessons/) 
+includes an introduction to data, the UNIX shell, OpenRefine, and Git. 
+The extended Library Carpentry curriculum includes 
+SQL, webscraping, Python and an introduction to data for archivists.
```
Comment (amyehodge) [inline _episodes/21-carpentries.md]: I'm not sure what is meant here by "extended."

File: _episodes/21-carpentries.md (hunk 1 above)
Comment (raynamharris) [reply _episodes/21-carpentries.md]: good catch!

File: _episodes/21-carpentries.md (hunk 3 above)
Comment (raynamharris) [reply _episodes/21-carpentries.md]: correct!

File: _episodes/21-carpentries.md (hunk 5 above)
Comment (raynamharris) [reply _episodes/21-carpentries.md]: 👍

File: _episodes/21-carpentries.md (hunk 6 above)
Comment (raynamharris) [reply _episodes/21-carpentries.md]: good catch!

File: _episodes/21-carpentries.md (hunk 4 above)
Comment (raynamharris) [reply _episodes/21-carpentries.md]: correct. great suggestions

File: _episodes/21-carpentries.md (hunk 2 above)
Comment (raynamharris) [reply _episodes/21-carpentries.md]: good catch!

File: _episodes/21-carpentries.md (hunk 7 above)
Comment (raynamharris) [reply _episodes/21-carpentries.md]: corrected!

File: _episodes/21-carpentries.md (hunk 8 above)
Comment (raynamharris) [reply _episodes/21-carpentries.md]: in addition to the core or underdevelopment. this has been clarified.

**Comment by raynamharris** [comment]
Thanks @amyehodge I incorporated all your suggestions and clarified some wording!

**Comment by amyehodge** [comment]
We also need to include Library Carpentry in the checkout instructions at http://carpentries.github.io/instructor-training/checkout/. Should that go in this PR or should I make a new issue?

**Comment by raynamharris** [comment]
Excellent point, @amyehodge. I'll go ahead and try to find all instances where LC should be added and incorporate them here.

File: _episodes/21-carpentries.md (hunk 8 above)
Comment (karenword) [reply _episodes/21-carpentries.md]: I think "almost" should be "also"?

**Comment by karenword** [comment]
This is awesome -- thanks so much @raynamharris !

**Comment by libcce** [comment]
Thanks @raynamharris for doing this! I've submitted a PR https://github.com/carpentries/instructor-training/pull/820 for changes that I felt comfortable suggesting. The other suggestions/questions I have include:

- Add new Venn diagram based on https://github.com/LibraryCarpentry/governance/issues/8 (style option 2)
- LC is not trademarked, Software and Data Carpentry are, but it is on our staff list of tasks
- There is now a unified "Request a Carpentries Workshop" form https://amy.software-carpentry.org/forms/workshop/. Should we collapse the request workshops links under 'How a Workshop Works' into one form for all the Lesson Programs?
- Under 'Who Can Teach What', certified instructors can teach all the Lesson Programs. What language should we use?

**Comment by raynamharris** [comment]
Thanks @libcce. Should be open a new issue with some of the pending things so they can be addressed with new PRs when the time is right?

**Comment by ChristinaLK** [comment]
I've created an issue.

**Pull request merged**